### DISCUSSION TITLE
Can I specify "Initial Jump Index (Position)" for a snippet?

### DISCUSSION THREAD

**Discussion by ziontee113** [L3MON4D3/LuaSnip#633] [Q&A] [answered]
Hi, I wonder if it's possible to set up the initial jump index after triggering a snippet.
Let's say after triggering the snippet, I'd like to jump to the `0` index for example.

Can I do this in the current version of LuaSnip? Thank you very much!

**Comment by L3MON4D3** [comment, marked as answer]
Yes! #562

**Reply by ziontee113** [reply]
Thank you very much! It worked with any id :smile:
